xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/PropAddAction.java

```diff
@@ -80,13 +84,19 @@ public boolean action(XWikiContext context) throws XWikiException
         BaseClass bclass = doc.getXClass();
         bclass.setName(doc.getFullName());
         if (bclass.get(propName) != null) {
-            context.put("message", "action.addClassProperty.error.alreadyExists");
+            context.put("message", PROPERTY_ERROR_ALREADY_EXISTS_ERROR_MESSAGE);
             List<String> parameters = new ArrayList<String>();
             parameters.add(propName);
             context.put("messageParameters", parameters);
```

Comment:
I don't understand why you're still adding the message and the parameters to the context. Has this any use when the action is aborted?

Reply:
Good point, the context is not used afterward when false is returned, I can remove that code.